### PR TITLE
Support for "okteto down -v"

### LINKED ISSUE #533: Support for "okteto down -v"
**Is your feature request related to a problem? Please describe.**
Kubernetes does not apply `fsGroup` to existing folders in a PVC. In order to change `fsGroup` in the okteto manifest, the user needs to run `okteto down` for every active dev environment,  delete the okteto volume, and run `okteto up` again.

**Describe the solution you'd like**
`okteto volume -v` removes the folder of the current dev environment in the okteto volume.

### PR DESCRIPTION
**Pull request by pchico83**
Fixes #533

### CODE REVIEW

File: cmd/down.go [hunk 1]
```diff
@@ -44,6 +46,14 @@ func Down() *cobra.Command {
 
 			log.Success("Development environment deactivated")
 
+			if rm {
+				if err := cleanVolume(dev); err != nil {
```
Comment (rberrelleza) [inline cmd/down.go]: we should create a separate event for this specifically, so we know if the clean fails vs the down operation.

File: pkg/k8s/pods/pod.go [hunk 2]
```diff
@@ -178,6 +178,54 @@ func Restart(dev *model.Dev, c *kubernetes.Clientset) error {
 	return waitUntilRunning(dev.Namespace, fmt.Sprintf("%s=%s", okLabels.DetachedDevLabel, dev.Name), c)
 }
 
+// RunCleanerPod runs a pod to clean the dev environmentt volume
+func RunCleanerPod(dev *model.Dev, c *kubernetes.Clientset) error {
```
Comment (rberrelleza) [inline pkg/k8s/pods/pod.go]: It feels like a `job` would be more suitable for this than the pod no?

File: pkg/k8s/pods/translate.go [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+)
+
+const (
+	oktetoVolumeName = "okteto"
+)
+
+func translate(dev *model.Dev) *apiv1.Pod {
+	return &apiv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("volume-cleaner-%s", dev.Name),
```
Comment (rberrelleza) [inline pkg/k8s/pods/translate.go]: if the user runs `okteto down -v` multiple times could this be problematic?

File: cmd/down.go [hunk 4]
```diff
@@ -148,6 +159,34 @@ func removeVolumes(dev *model.Dev) error {
...
+	if dev.Namespace == "" {
+		dev.Namespace = namespace
+	}
+
+	exists, err := volumes.Exists(dev.Namespace, client)
+	if err != nil {
+		return err
+	}
+
+	if !exists {
```
Comment (rberrelleza) [inline cmd/down.go]: in which situations it wouldn't exist?

File: cmd/up.go [hunk 5]
```diff
@@ -434,7 +434,7 @@ func (up *UpContext) startLocalSyncthing() error {
 	if err := up.Sy.WaitForPing(up.Context, false); err != nil {
 		return errors.UserError{
 			E:    fmt.Errorf("Failed to connect to the synchronization service"),
-			Hint: fmt.Sprintf("If you are using a non-root container, you need to set the securityContext.runAsUser and securityContext.runAsGroup values in your Okteto manifest (https://okteto.com/docs/reference/manifest/index.html#securityContext-object-optional)."),
+			Hint: fmt.Sprintf("If you are using a non-root container, run 'okteto down -v' and set the fields securityContext.runAsUser, securityContext.runAsGroup and securityContext.fsGroup in your Okteto manifest (https://okteto.com/docs/reference/manifest/index.html#securityContext-object-optional)."),
```
Comment (rberrelleza) [inline cmd/up.go]:
```suggestion
			Hint: fmt.Sprintf("If you are using a non-root container, run 'okteto down -v' to clean the existing files, set the securityContext.runAsUser, securityContext.runAsGroup and securityContext.fsGroup fields in your Okteto manifest, and try again (https://okteto.com/docs/reference/manifest/index.html#securityContext-object-optional)."),
```

File: cmd/down.go (hunk 4 above)
Comment (pchico83) [reply cmd/down.go]: the user can remove it manually

File: pkg/k8s/pods/pod.go (hunk 2 above)
Comment (pchico83) [reply pkg/k8s/pods/pod.go]: Not really. The job would create a pod, and we would need to monitor the pod anyway, and clean the pod and the job. jobs are suitable for cronjobs or tasks where you want to run up to 1000 tasks with less than X errors.

**Pull request merged**